<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Status</class>
 <widget class="QWidget" name="Status">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Status</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Status {
    background: #1a1412;
}

QLabel {
    background: transparent;
    font-family: &quot;Press Start 2P&quot;, &quot;Courier New&quot;, monospace;
}

QGroupBox {
    background: transparent;
    border: 2px solid #2a1d19;
    border-radius: 0px;
    margin-top: 16px;
    padding-top: 12px;
    padding-left: 12px;
    font-size: 10px;
    letter-spacing: 0px;
    text-transform: uppercase;
    color: #8d7b68;
    font-weight: bold;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 8px;
    top: -8px;
    padding: 0 4px 0 4px;
    background: #1a1412;
}

QScrollArea {
    background: transparent;
    border: none;
}

QScrollBar:vertical {
    background: #14100f;
    width: 12px;
    border: 2px solid #2a1d19;
    border-radius: 0px;
}

QScrollBar::handle:vertical {
    background: #3d2b25;
    border: 1px solid #5a4339;
    border-radius: 0px;
    min-height: 30px;
}

QScrollBar::handle:vertical:hover {
    background: #5a4339;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    height: 0px;
}

QTableWidget {
    background: #14100f;
    border: 1px solid #2a1d19;
    color: #e8d5c4;
    gridline-color: #2a1d19;
    selection-background-color: #3d2b25;
    selection-color: #c4a77d;
}

QTableWidget::item {
    padding: 8px;
}

QTableWidget::item:selected {
    background: #3d2b25;
    color: #c4a77d;
}

QHeaderView::section {
    background: #2a1d19;
    color: #c4a77d;
    padding: 8px;
    border: none;
    border-right: 1px solid #3d2b25;
    border-bottom: 1px solid #3d2b25;
    font-weight: bold;
}

QProgressBar {
    background: #14100f;
    border: 1px solid #2a1d19;
    border-radius: 0px;
    text-align: center;
    color: #c4a77d;
}

QProgressBar::chunk {
    background: #3d2b25;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="contentWidget">
     <property name="styleSheet">
      <string notr="true">QWidget#contentWidget {
    background: #1a1412;
}</string>
     </property>
     <layout class="QVBoxLayout" name="contentLayout">
      <property name="spacing">
       <number>24</number>
      </property>
      <property name="leftMargin">
       <number>48</number>
      </property>
      <property name="topMargin">
       <number>40</number>
      </property>
      <property name="rightMargin">
       <number>48</number>
      </property>
      <property name="bottomMargin">
       <number>40</number>
      </property>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>SYSTEM STATUS</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #c4a77d;
    font-size: 18px;
    font-weight: bold;
    letter-spacing: 1px;
    background: transparent;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QScrollArea {
    border: none;
}</string>
        </property>
        <widget class="QWidget" name="scrollContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>820</width>
           <height>520</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="scrollLayout">
          <property name="spacing">
           <number>24</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="systemInfoGroup">
            <property name="title">
             <string>SYSTEM INFORMATION</string>
            </property>
            <layout class="QVBoxLayout" name="systemInfoLayout">
             <property name="spacing">
              <number>12</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>12</number>
             </property>
             <item>
              <widget class="QTableWidget" name="systemInfoTable">
               <property name="minimumHeight">
                <number>200</number>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::NoSelection</enum>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <column>
                <property name="text">
                 <string>Property</string>
                </property>
                <property name="width">
                 <number>200</number>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Value</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="statisticsGroup">
            <property name="title">
             <string>USAGE STATISTICS</string>
            </property>
            <layout class="QVBoxLayout" name="statisticsLayout">
             <property name="spacing">
              <number>12</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>12</number>
             </property>
             <item>
              <widget class="QTableWidget" name="statisticsTable">
               <property name="minimumHeight">
                <number>200</number>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::NoSelection</enum>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <column>
                <property name="text">
                 <string>Property</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Value</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="cacheGroup">
            <property name="title">
             <string>CACHE MANAGEMENT</string>
            </property>
            <layout class="QVBoxLayout" name="cacheLayout">
             <property name="spacing">
              <number>12</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>12</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="cacheInfoLayout">
               <item>
                <widget class="QLabel" name="cacheSizeLabel">
                 <property name="text">
                  <string>Total Cache Size: Scanning...</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: #c4a77d;
    font-size: 11px;
    background: transparent;</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="cacheInfoSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="btnRefreshCache">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>28</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>REFRESH</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    background: #1a1412;
    color: #c4a77d;
    border: 2px solid #3d2b25;
    border-radius: 0px;
    font-size: 11px;
    font-weight: bold;
    padding: 4px 12px;
}

QPushButton:hover {
    background: #3d2b25;
    color: #e8d5c4;
}

QPushButton:pressed {
    background: #c4a77d;
    color: #1a1412;
}</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTableWidget" name="cacheTable">
               <property name="minimumHeight">
                <number>150</number>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <column>
                <property name="text">
                 <string>Cache Name</string>
                </property>
                <property name="width">
                 <number>150</number>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Size</string>
                </property>
                <property name="width">
                 <number>100</number>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Files</string>
                </property>
                <property name="width">
                 <number>80</number>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Description</string>
                </property>
               </column>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="cacheButtonsLayout">
               <property name="spacing">
                <number>12</number>
               </property>
               <item>
                <widget class="QPushButton" name="btnClearSelected">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>CLEAR SELECTED</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    background: #3d2b25;
    color: #e8d5c4;
    border: 2px solid #5a4339;
    border-radius: 0px;
    font-size: 11px;
    font-weight: bold;
    padding: 6px 12px;
}

QPushButton:hover {
    background: #5a4339;
    color: #c4a77d;
}

QPushButton:pressed {
    background: #c4a77d;
    color: #1a1412;
}

QPushButton:disabled {
    background: #1a1412;
    color: #6b5c52;
    border-color: #2a1d19;
}</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnClearAll">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>CLEAR ALL CACHE</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    background: #5a4339;
    color: #e8d5c4;
    border: 2px solid #8d7b68;
    border-radius: 0px;
    font-size: 11px;
    font-weight: bold;
    padding: 6px 12px;
}

QPushButton:hover {
    background: #8d7b68;
    color: #c4a77d;
    border-color: #c4a77d;
}

QPushButton:pressed {
    background: #c4a77d;
    color: #1a1412;
    border-color: #c4a77d;
}</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="cacheButtonsSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="bottomSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
